src/makeCalculator.js

```diff
@@ -37,7 +37,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value) {
+      this.result += value;
+
+      return this;
+    },
+    subtract(value) {
+      this.result -= value;
+
+      return this;
+    },
+    multiply(value) {
+      this.result *= value;
+
+      return this;
+    },
+    divide(value) {
+      if (value > 0) {
```

Comment:
It's incorrect to perform division operation only when the value is greater than 0. Division by 0 is undefined but not illegal, and it's better to handle that case separately and notify the user about it instead of silently ignoring division by 0.